apps/web/src/views/Voting/components/CastVoteModal/DetailsView.tsx

```diff
@@ -105,9 +105,7 @@ const DetailsView: React.FC<React.PropsWithChildren<DetailsViewProps>> = ({
             {`${t('CAKE locked:')} ${formatNumber(lockedCakeBalance, 0, 2)}`}
           </Text>
           <NextLinkFromReactRouter to="/pools" prefetch={false}>
-            <Link href="/pools" color="primary">
-              {t('Go to Pools')}
-            </Link>
+            <StyledLink color="primary">{t('Go to Pools')}</StyledLink>
```

Comment:
Do you think when we clicked `Go to Pools` switch to `/pools` page we should close the `CastVoteModal` ?

Reply:
![Screenshot 2023-09-04 at 12 56 54 PM](https://github.com/pancakeswap/pancake-frontend/assets/98292246/df3e66fa-e666-4497-b2f9-0fe96af8eb20)

Reply:
Change it, when click open new page to `/pools`